server/src/lib/model/db.ts

```diff
@@ -721,6 +719,14 @@ export default class DB {
     )[0][0];
   }
 
+  async findSentence(id: string) {
+    return (
+      await this.mysql.query('SELECT * FROM sentences WHERE id = ? LIMIT 1', [
```

Comment:
Nitpick - since we only need the `locale_id` and the `text`, we should specify those two columns instead of grabbing everything. I know most of the other columns are pretty small, but since the sentences table is in our top 3 biggest tables, the less we have to scan through the better.

Reply:
Fixed in latest commit ✅